manifests/init.pp

```diff
@@ -34,14 +34,18 @@
   String  $fipscheck_ensure = simplib::lookup('simp_options::package_ensure', { 'default_value' => 'installed' }),
   String  $nss_ensure       = simplib::lookup('simp_options::package_ensure', { 'default_value' => 'installed' })
 ) {
-
   simplib::assert_metadata($module_name)
 
   $fips_kernel_value = $enabled ? {
     true    => '1',
     default => '0'
   }
 
+  $fipscheck_package_name = $facts['os']['release']['major'] > 8 ? {
```

Comment:
Looks like this can result in a String vs Integer comparison (compilation error): 
https://github.com/simp/pupmod-simp-fips/actions/runs/3952768811/jobs/6768251257#step:5:38

Reply:
```suggestion
  $fipscheck_package_name = "${facts['os']['release']['major']}" > '8' ? {
```

Reply:
Went a different direction than setting a variable like this. Module-level hieradata will determine the package name and allow users to override the package name if necessary.